exs/mundo_1/javascript/035.js

```diff
@@ -0,0 +1,17 @@
+const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados
+
+//Entrada de dados
+let a = input("Primeiro segmento de reta: \n")
+let b = input("Segundo segmento de reta: \n")
+let c = input("Terceiro segmento de reta: \n")
+
+//Condição if e else para saber se as partes formam um triangulo
+if (a < b + c && b < a + c && c + a  > b){
+    console.log("O triângulo existe.")
+}
+
+else {
+    console.log("O triângulo não pode existir.")
+}
+
+//O operador || significa "OU" e o operador && siginfica e ou and 
```

Comment:
Correção ortográfica e de explicação necessária 

```suggestion
//O operador || significa "OU" e o operador && significa "E"
```